luaui/Widgets/gui_chat.lua

```diff
@@ -2093,7 +2093,7 @@ function widget:MapDrawCmd(playerID, cmdType, x, y, z, a, b, c)
 end
 
 function widget:AddConsoleLine(lines, priority)
-	lines = lines:match('^\[f=[0-9]+\] (.*)$') or lines
+	lines = lines:match('^\\[f=[0-9]+\\] (.*)$') or lines
```

Comment:
This looks wrong, looks like somebody was trying to match `[` as character, not match `\[` string. I assume this should be `%[`, that's how you escape special characters in regexp.

Reply:
Isn't `\` the escape character in regex? I'm not seeing any sources that have `%` as the escape character.

Reply:
https://lua.org/manual/5.1/manual.html#5.4.1

> %x: (where x is any non-alphanumeric character) represents the character x. This is the standard way to escape the magic characters. Any punctuation character (even the non magic) can be preceded by a '%' when used to represent itself in a pattern.

Reply:
Changed `\\` to `%`